olm-catalog/serverless-operator/Dockerfile

```diff
@@ -13,7 +13,7 @@ LABEL operators.operatorframework.io.bundle.channels.v1="4.5,4.6"
 LABEL \
```

Comment:
I'm curious if this file should target only OCP 4.6. The component matrix says 4.6 will be the only supported: https://docs.google.com/spreadsheets/d/1HTxR37_MM03_JQImW-KmAFk1oEQ83fXzfGxdfWQSX6o/edit#gid=0
However, current CI on 4.6 keeps failing due to issues with the index image.

Reply:
Maybe yes. Let's revisit after getting the initial bits in though to unblock some stuff that depends on the 0.17 bump.